cloud/edgecontroller/hack/boilerplate/boilerplate.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright YEAR The KubeEdge Authors.
```

Comment:
nit : YEAR -> 2019

Reply:
This is boilerplate template, when script runs it is replaced with current year.

Reply:
ok, thanks

Reply:
Do edge componets need the template? If yes, place it in a common place where can be shared by both edge and cloud components?

Reply:
this template is only for generated code. we don't have such case for edge.